src/dev/flang/tools/docs/Html.java

```diff
@@ -218,6 +220,68 @@ private static String source(AbstractFeature feature)
       .replace("$1", featureURL(feature));
   }
 
+  static String processComment(String name, String s)
```

Comment:
Can you add a comment explaining what this does, what the two parameters are and what the result will be?